### DISCUSSION TITLE
archives/740/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#17] [Announcements]
# archives/740/

最近在探索 m1 MacBook 下运行 linux 虚拟机的可能性，计划未来在 linux 虚拟机下完成所有开发工作，parallels desktop 天然提供了 debian/ubuntu/fedora 等 linux 发行版支持，但是在我个人使用

https://www.frytea.com/archives/740/

**Comment by dionysusandapollo** [comment]
可以安装hyprland吗
